docs/content/references/hfm.md

```diff
@@ -0,0 +1,114 @@
+[^kt]: LaTeX is rendered with [KaTeX][katex].
+
+## Links & Images
+
+| Feature               | HFM 1 | HFM 2 | CommonMark       | GFM              |
+|-----------------------|:-----:|:-----:|:----------------:|:----------------:|
+| link                  | ☑️     | ☑️     | ☑️                | ☑️                |
+| link reference        | ☑️     | ☑️     | ☑️                | ☑️                |
+| link title            | ☑️     | ☑️     | ☑️                | ☑️                |
+| autolink with `<>`    | ☑️     | ☑️     | ☑️                | ☑️                |
+| autolink without `<>` | ☑️     | ☑️     |                  | ☑️                |
+| footnotes             | ☑️     | ☑️     |                  |                  |
+| image                 | ☑️     | ☑️     | ☑️                | ☑️                |
+| image with given size | ☑️     | ☑️     | (☑️ with `<img>`) | (☑️ with `<img>`) |
+| table of contents     | ☑️     | ☑️     |                  |                  |
+
+
+## Structural elements
+
+| Feature           | HFM 1                    | HFM 2                    | CommonMark           | GFM                  |
+|-------------------|:------------------------:|:------------------------:|:--------------------:|:--------------------:|
+| table             | ☑️                        | ☑️                        | (☑️ with `<table>`)   | ☑️                    |
+| horizontal line   | ☑️                        | ☑️                        | ☑️                    | ☑️                    |
+| collapsable block | (☑️ with `<details>`)     | ☑️                        | (☑️ with `<details>`) | (☑️ with `<details>`) |
+| Alerts            | ☑️                        | ☑️                        |                      |                      |
+
+## Embeddings
+HFM1 included support for certain embeddings of external content. These were defined in markdown by the uncommon `{%keyword parameter %}` syntax.
+
+Instead of this uncommon syntax, HFM2 uses just plain links to external content and creates embeddings for supported providers.
+
+| Feature                  | HFM 1 | HFM 2              | CommonMark | GFM |
+|--------------------------|:-----:|:------------------:|:----------:|:---:|
+| PDF (`{%pdf ... %}`)     | ☑️     | removed            |            |     |
+| [YouTube][youtube] (`{%youtube ... %}`)       | ☑️     | with plain link[^embed] |            |     |
+| [Vimeo][vimeo] (`{%vimeo ... %}`)          | ☑️     | with plain link[^embed] |            |     |
+| [Slideshare][slideshare] (`{%slideshare ... %}`) | ☑️     | removed            |            |     |
+| [Speakerdeck][speakerdeck] (`{%speakerdeck ... %}`) | ☑️     | removed            |            |     |
+| [GitHub Gist][gist] (`{%gist ... %}`)             | ☑️     | with plain link[^embed] |            |     |
+
+[^embed]: The special syntax is deprecated but will continue to work. However a plain link to the content will generate the same embedding and is preferred.
```

Comment:
```suggestion
[^embed]: The special syntax from HFM 1 is deprecated but will still work in HFM 2. However a plain link to the content should be preferred.
```

Reply:
I incorporated this, but changed the last words to `should be used.` instead of `should be preferred`.

Reply:
How about making it must be preferred? This would make it clear that these might drop out of support any time in future.

Reply:
I think I like the current wording best